README.md

```diff
@@ -106,9 +106,11 @@ DEFAULT_INSTANCE_BASE_DOMAIN='example.com'
 GANDI_API_KEY='api-key'
 GITHUB_ACCESS_TOKEN='github-token'
 WATCH_ORGANIZATION='github-org'
-RABBITMQ_API_URL='https://rabbitmq-dev.opencraft.hosting:15671'
-RABBITMQ_ADMIN_USERNAME='admin'
-RABBITMQ_ADMIN_PASSWORD='changeme'
+DEFAULT_RABBITMQ_API_URL='https://rabbitmq-dev.opencraft.hosting:15671'
+DEFAULT_RABBITMQ_ADMIN_USERNAME='admin'
+DEFAULT_RABBITMQ_ADMIN_PASSWORD='changeme'
+DEFAULT_INSTANCE_RABBITMQ_HOST='rabbitmq-dev.opencraft.hosting'
```

Comment:
@pomegranited Yeah actually I can refactor these quite easily now to just use standard URIs, thanks for pointing that out! The result would be two different variables. I could move `DEFAULT_INSTANCE_RABBITMQ_HOST` and `DEFAULT_INSTANCE_RABBITMQ_PORT` into a single variable containing an [AMQP URI](https://www.rabbitmq.com/uri-spec.html): `amqp://rabbitmq-dev.opencraft.hosting:5671`

And the API uses basic auth, so I can just move the username and password right into the `DEFAULT_RABBITMQ_API_URL`: `https://admin:changeme@rabbitmq-dev.opencraft.hosting:15671`